be/src/exec/schema_scanner/schema_partitions_scanner.cpp

```diff
@@ -75,21 +77,14 @@ Status SchemaPartitionsScanner::start(RuntimeState* state) {
     }
     SCOPED_TIMER(_get_db_timer);
     TGetDbsParams db_params;
-    if (nullptr != _param->common_param->db) {
+    if (_param->common_param->db) {
         db_params.__set_pattern(*(_param->common_param->db));
     }
-    if (nullptr != _param->common_param->catalog) {
+    if (_param->common_param->catalog) {
         db_params.__set_catalog(*(_param->common_param->catalog));
     }
-    if (nullptr != _param->common_param->current_user_ident) {
+    if (_param->common_param->current_user_ident) {
         db_params.__set_current_user_ident(*(_param->common_param->current_user_ident));
-    } else {
-        if (nullptr != _param->common_param->user) {
-            db_params.__set_user(*(_param->common_param->user));
-        }
```

Comment:
why  remove these logic

Reply:
user and user_ip params are deprecated in TGetDbsParams so removed it.